lib/sparql.jison

```diff
@@ -13,9 +13,7 @@
       XSD_INTEGER  = XSD + 'integer',
       XSD_DECIMAL  = XSD + 'decimal',
       XSD_DOUBLE   = XSD + 'double',
-      XSD_BOOLEAN  = XSD + 'boolean',
-      XSD_TRUE =  '"true"^^'  + XSD_BOOLEAN,
-      XSD_FALSE = '"false"^^' + XSD_BOOLEAN;
```

Comment:
You might want to make all of these already `NamedNode`s.

Reply:
But `createLiteral()` second arg is string, not `Term`. I don't think we should treat these specially.

Reply:
> But createLiteral() second arg is string

That's a very good reason to make that second arg a named node, because it avoids re-creating all those XSD nodes again and again.